config/main.py

```diff
@@ -564,7 +596,12 @@ def startup(interface_name, verbose):
 def speed(interface_name, interface_speed, verbose):
     """Set interface speed"""
     if get_interface_mode() == "alias": 
-        interface_name = interface_alias(interface_name)
+        default_name = interface_alias_to_name(interface_name)
```

Comment:
Please refactor this code in the same manner as I suggested in my comment on line 411.